From c5529352e1edcbe8a0be05f7083ee8733a2aff0a Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Mon, 15 Jan 2018 16:30:43 -0800
Subject: [PATCH 0751/1239] configs: t93: remove EFI_HELLO_WORLD example efi
 app

EFI_SUBSYSTEM used in coff header in crt0_aarch64_efi.S creates
compilation error with newest binutils.
Newest binutils rejects creating a shared library with dynamic
relocation of R_AARCH64_ABS16 as there is no support by glibc
for that dynamic relocation. This needs fix in mainline too.
Disabling helloworld app compile until it is fixed in mainline.

Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
Reported-by: Andrew Pinski <apinski@cavium.com>
---
 configs/t93_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/t93_defconfig b/configs/t93_defconfig
index 9f0ab2e22c..5215e8bcf8 100644
--- a/configs/t93_defconfig
+++ b/configs/t93_defconfig
@@ -19,6 +19,7 @@ CONFIG_BOARD_EARLY_INIT_R=y
 CONFIG_HUSH_PARSER=y
 CONFIG_SYS_PROMPT="Cavium> "
 # CONFIG_AUTOBOOT is not set
+# CONFIG_CMD_BOOTEFI_HELLO_COMPILE is not set
 CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_MX_CYCLIC=y
 # CONFIG_CMD_FLASH is not set
-- 
2.29.0

